framework/JobHandler.py

```diff
@@ -672,8 +681,17 @@ def cleanJobQueue(self):
           ## it by calling numRunning.
           with self.__queueLock:
             self.__finished.append(run)
+            self.__finished[-1].trackTime('jobHandler_finished')
             runList[i] = None
 
+  def setProfileJobs(self,profile=True):
```

Comment:
shouldn't be ```False``` by default?

Reply:
My thinking was that if the method is being called, then you mean to set it to "True"; I can easily reverse this though.